inlang/source-code/marketplace-registry/registry.json

```diff
@@ -1,15 +1,15 @@
-	"https://cdn.jsdelivr.net/gh/inlang/monorepo@latest/inlang/source-code/badge/marketplace-manifest.json",
-	"https://cdn.jsdelivr.net/gh/inlang/monorepo@latest/inlang/source-code/plugins/json/marketplace-manifest.json",
-	"https://cdn.jsdelivr.net/gh/inlang/monorepo@latest/inlang/source-code/plugins/i18next/marketplace-manifest.json",
-	"https://cdn.jsdelivr.net/gh/inlang/monorepo@latest/inlang/source-code/message-lint-rules/emptyPattern/marketplace-manifest.json",
-	"https://cdn.jsdelivr.net/gh/inlang/monorepo@latest/inlang/source-code/message-lint-rules/identicalPattern/marketplace-manifest.json",
-	"https://cdn.jsdelivr.net/gh/inlang/monorepo@latest/inlang/source-code/message-lint-rules/messageWithoutSource/marketplace-manifest.json",
-	"https://cdn.jsdelivr.net/gh/inlang/monorepo@latest/inlang/source-code/message-lint-rules/missingTranslation/marketplace-manifest.json",
-	"https://cdn.jsdelivr.net/gh/inlang/monorepo@latest/inlang/source-code/versioned-interfaces/language-tag/marketplace-manifest.json",
-	"https://cdn.jsdelivr.net/gh/inlang/monorepo@latest/inlang/source-code/versioned-interfaces/translatable/marketplace-manifest.json",
-	"https://cdn.jsdelivr.net/gh/inlang/monorepo@latest/inlang/source-code/paraglide-js-sveltekit/marketplace-manifest.json"
+	"https://raw.githubusercontent.com/floriandwt/inlang-marketplace-test/main/badge/marketplace-manifest.json",
+	"https://raw.githubusercontent.com/floriandwt/inlang-marketplace-test/main/cli/marketplace-manifest.json",
+	"https://raw.githubusercontent.com/floriandwt/inlang-marketplace-test/main/editor/marketplace-manifest.json",
+	"https://raw.githubusercontent.com/floriandwt/inlang-marketplace-test/main/guide/marketplace-manifest.json",
+	"https://raw.githubusercontent.com/floriandwt/inlang-marketplace-test/main/ide-extension/marketplace-manifest.json",
+	"https://raw.githubusercontent.com/floriandwt/inlang-marketplace-test/main/message-lint-rules/emptyPattern/marketplace-manifest.json",
+	"https://raw.githubusercontent.com/floriandwt/inlang-marketplace-test/main/message-lint-rules/identicalPattern/marketplace-manifest.json",
+	"https://raw.githubusercontent.com/floriandwt/inlang-marketplace-test/main/message-lint-rules/messageWithoutSource/marketplace-manifest.json",
+	"https://raw.githubusercontent.com/floriandwt/inlang-marketplace-test/main/message-lint-rules/missingTranslation/marketplace-manifest.json",
+	"https://raw.githubusercontent.com/floriandwt/inlang-marketplace-test/main/plugins/i18next/marketplace-manifest.json",
+	"https://raw.githubusercontent.com/floriandwt/inlang-marketplace-test/main/plugins/json/marketplace-manifest.json",
+	"https://raw.githubusercontent.com/floriandwt/inlang-marketplace-test/main/versioned-interfaces/language-tag/marketplace-manifest.json",
+	"https://raw.githubusercontent.com/floriandwt/inlang-marketplace-test/main/versioned-interfaces/translatable/marketplace-manifest.json"
```

Comment:
`raw.githubusercontent.com` is blocked in China and a few other countries. `cdn.jsdelivr.net` works sometimes better.

Reply:
@openscript thank you for the information! :) Didn't know that the `github cdn` is blocked in China too, but these are just links to my own test repo which will get changed before merging anyways.